Src/Witsml/Query/TubularQueries.cs

```diff
@@ -0,0 +1,34 @@
+using Witsml.Data;
+using Witsml.Extensions;
+
+namespace Witsml.Query
+{
+    public class TubularQueries
```

Comment:
There is already a discussion in #350 when it comes to query code and I suspect that such code will be moved to the API project. I propose to leave this out of the PR for now

Reply:
Removed tubular queries from Witsml library. Instead building the query where they are used.